src/domain/synthetics/markets/useGlvMarkets.ts

```diff
@@ -96,12 +100,14 @@ export function useGlvMarketsInfo(
         };
```

Comment:
regarding `GlvMarketInfo extends MarketInfo`
could you explain what exactly take from marketInfo?
I assume they shouldn't share very much in common.

I'd suggest to elliminate this inheritance, also rename GlvMarketInfo to GlvInfo

Reply:
Completely separated GlvInfo from MarketInfo